TouchEffect/Delegates/TEffectHoverStateChangedHandler.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using TouchEffect.EventArgs;
+using Xamarin.Forms;
+
+namespace TouchEffect.Delegates
+{
+    [Obsolete]
+    public delegate void TouchViewHoverStateChangedHandler(TouchView sender, HoverStateChangedEventArgs args);
```

Comment:
That's a little weird to add obsolete API :)
please remove obsolete code at all (don't add it)